symbolic_trace/infer_meta.py

```diff
@@ -118,7 +112,19 @@ def convert_to_input_spec(args):
     )
 
 
-@no_eval_frame
+def variable_to_meta_info(args):
+    return map_if(
+        args,
+        pred=lambda x: isinstance(x, paddle.static.Variable),
+        true_fn=lambda x: MetaInfo(
+            list(x.shape),
+            x.dtype,
+            x.stop_gradient,
+        ),
+        false_fn=lambda x: paddle.static.InputSpec.from_tensor(x),
```

Comment:
```suggestion
        false_fn=lambda x: MetaInfo.from_tensor(x),
```
这里应该是MetaInfo.from_tensor ?  不然 true-fn 和 false-fn 返回的类型不一致